Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2_test.py

```diff
@@ -1520,3 +1522,22 @@ def test_update_remote_system_command(mocker, prisma_cloud_v2_mirroring_client,
 
     assert mock_update_remote_alert.call_count == expected_call_count
     assert result == 'P-1111111'
+
```

Comment:
add more broad test where passing, include_resource_json=false will return response without the two additional fields.

Reply:
The current test tests both remove and non-removed fields.
Testing the calling method means using mocking, which will not test my new code.